### PR TITLE
added new workflow for managing memory store

### PR DESCRIPTION
**Pull request by maxsap**

### CODE REVIEW

**Review by RaulEscobarRivas — APPROVED** [review]

File: .github/workflows/memory_store.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            --rdb-snapshot-period=${{ inputs.SNAPSHOT_PERIOD }} \
+            --connect-mode=direct-peering  
+          else
+            gcloud redis instances update ${{ inputs.CLUSTER_NAME }} --region=${{ inputs.REGION }} \
+            --rdb-snapshot-period=${{ inputs.SNAPSHOT_PERIOD }} \
+            --replica-count=${{ inputs.REPLICA_COUNT }} \
+            --size=${{ inputs.MEMORY }}
+          fi
+          
+          echo "::set-output name=url::$(echo $clusterDesc | jq -r '"\(.readEndpoint):\(.readEndpointPort)"')"
```
Comment (DMalliaros) [inline .github/workflows/memory_store.yml]: Is `jq` part of github action?

File: .github/workflows/cloud_run.yml [hunk 2]
```diff
@@ -109,6 +115,21 @@ jobs:
...
+            emptyResponse=$(echo "[]")
+            connectorListResult=$(gcloud compute networks vpc-access connectors list --region=${{ inputs.REGION }} --filter="projects/${{ secrets.PROJECT_ID }}/locations/${{ inputs.REGION }}/connectors/${{ secrets.PROJECT_ID }}-vpc-connector" --format=json)
+            if [ $connectorListResult == $emptyResponse ]; then
+              gcloud compute networks vpc-access connectors create ${{ secrets.PROJECT_ID }}-vpc-connector \
+                --network default \
+                --region ${{ inputs.REGION }} \
+                --range 10.8.0.0/28
+            fi
+
+            defaultFlags=$(echo "$defaultFlags" --vpc-connector=${{ secrets.PROJECT_ID }}-vpc-connector)
```
Comment (DMalliaros) [inline .github/workflows/cloud_run.yml:130]: If you put the server in the VPC will be able to be accessed by frontend?

Comment (maxsap) [reply .github/workflows/cloud_run.yml:130]: yes

File: .github/workflows/memory_store.yml (hunk 1 above)
Comment (maxsap) [reply .github/workflows/memory_store.yml]: I am not sure, but I can change to `value`

**Review by DMalliaros — APPROVED** [review]

**Pull request merged**